python/cudf/cudf/core/dtypes.py

```diff
@@ -905,9 +905,12 @@ def __init__(self, subtype, closed="right"):
     def subtype(self):
         return self.fields["left"]
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return f"interval[{self.subtype}, {self.closed}]"
 
+    def __str__(self) -> str:
```

Comment:
Just noting that pandas' `IntervalDtype.__str__` considers a few more cases, but I am not sure if cudf has all these cases implemented 

https://github.com/pandas-dev/pandas/blob/6d021ac5b51c879eb6e4409711f840ea170e5a1f/pandas/core/dtypes/dtypes.py#L1315-L1321

Reply:
In case of `cudf`, looks like `subtype` is mandatory and optional for pandas.